app/main/views/sign_in.py

```diff
@@ -157,8 +156,9 @@ def _do_login_dot_gov():  # $ pragma: no cover
             current_app.logger.info(f"activating user {usr.id} #notify-admin-1505")
             activate_user(usr.id)
         except BaseException as be:  # noqa B036
-            current_app.logger.exception(f"Error signing in: {be} #notify-admin-1505 ")
+            current_app.logger.error(f"Error signing in: {be} #notify-admin-1505 ")
```

Comment:
Nice catch, it looks like this one was missed from the others that were flipped to `error`!

Reply:
If you change this to exception, you don't need the {be}.

Reply:
@xlorepdarkhelm changed the other lines to be `error` instead of `exception` so I'm assuming this one was meant to be changed as well and was just missed.  Thanks for the heads up though with the use of `exception`!